files/jquery.prettycheckable/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "prettyCheckable",
+  "repo": "arthurgouveia/prettyCheckable",
+  "files": {
+    "include": ["./img/*", "./dist/*", "./dev/*"]
```

Comment:
`"include": ["./img/*", "dist/*"]`

Also you can remove the files in this PR, the bot will add them automatically.